src/browser/browser.js

```diff
@@ -4,7 +4,7 @@
   'use strict';
 
   const {html, node, render, cache} = require('reflex');
-  const {Record, Any, Union, Maybe} = require('../common/typed');
+  const {Record, Maybe} = require('typed-immutable');
```

Comment:
Did a dependency update? Does this also need to change in all the other modules?